lib/modules/apostrophe-workflow-admin-bar/index.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
         return result;
       }
       var notSafeLiveList = [ 'apostrophe-pages', 'apostrophe-tags', 'apostrophe-workflow-manage-modal' ];
```

Comment:
Our pattern in other places and modules has been "baseNotSafeLiveList" and "notSafeLiveList," the latter being the one that gets concatenated to the base. The base gets set up as an option also, so that if someone is really on fire they may override it.

Reply:
Also in yesterday's engineering huddle we agreed to use the terms "allowList" and "denyList" in such contexts (where each is actually present - we didn't say both had to exist).